src/nyx/features/pixel.h

```diff
@@ -52,7 +52,8 @@ struct Pixel2 : public Point2i
 	Pixel2 (StatsInt x_, StatsInt y_, PixIntens i_) : Point2(x_, y_), inten(i_) {}
 	Pixel2 (int x_, int y_, PixIntens i_) : Point2(x_, y_), inten(i_) {}
 	Pixel2 (float x_, float y_, PixIntens i_) : Point2((StatsInt)x_, (StatsInt)y_), inten(i_) {}
-	Pixel2 (size_t x_, size_t y_, double i_) : Point2(x_, y_), inten(i_) {}
+	Pixel2 (size_t x_, size_t y_, double i_) : Point2((StatsInt)x_, (StatsInt)y_), inten((PixIntens)i_) {}

```

Comment:
Guessing the loss of precision from `i_` to `inten` is intentional.

Reply:
Yes, coordinates are 'unsigned int' anyway.